spec/features/chitter_spec.rb

```diff
@@ -1,15 +1,25 @@
 require "capybara/rspec"
+require "capybara"
 
 feature "it can call the website" do
   #   scenario "it can call the index page" do
   #     visit "/"
   #     expect(page).to have_content ("Hello World")
   #   end
 
-  scenario "it can sign a user in" do
+  #   scenario "it can sign a user in" do
+  #     visit("/")
+  #     fill_in :username, with: "Sarah"
+  #     click_button "Submit"
+  #     expect(page).to have_content "Sarah"
+  #   end
+
+  scenario "it can allow user to make a peep" do
     visit("/")
     fill_in :username, with: "Sarah"
     click_button "Submit"
-    expect(page).to have_content "Sarah"
+    fill_in :peep, with: "Hello World"
+    click_button "Submit"
+    expect(page).to have_content "Hello World"
```

Comment:
Contrary to your commit message, this doesn't test for db access - it's a feature test, testing that it works from the user's point of view. I think you passed this test by using parameters, not db access.